docs/en/multi_modal/gemma3.md

```diff
@@ -0,0 +1,30 @@
+# Gemma3
+
+## Introduction
+
+Gemma is a family of lightweight, state-of-the-art open models from Google, built from the same research and technology used to create the Gemini models. Gemma 3 models are multimodal, handling text and image input and generating text output, with open weights for both pre-trained variants and instruction-tuned variants. Gemma 3 has a large, 128K context window, multilingual support in over 140 languages, and is available in more sizes than previous versions. Gemma 3 models are well-suited for a variety of text generation and image understanding tasks, including question answering, summarization, and reasoning. Their relatively small size makes it possible to deploy them in environments with limited resources such as laptops, desktops or your own cloud infrastructure, democratizing access to state of the art AI models and helping foster innovation for everyone.
+
+## Quick Start
+
+Install LMDeploy by following the [installation guide](../get_started/installation.md).
+
+### Prepare
+
+When deploying the **Gemma3** model using LMDeploy, please install the latest transformers.
```

Comment:
may need to transformers from source like `pip install git+https://github.com/huggingface/transformers@v4.49.0-Gemma-3`  as https://huggingface.co/google/gemma-3-27b-it#usage

Reply:
It is merged into the main branch. Pulling the latest transformers is enough.